app/models/order.rb

```diff
@@ -5,4 +5,8 @@ class Order < ApplicationRecord
   has_many :books, through: :order_items
 
   has_one :coupon, dependent: :destroy
+
+  enum status: { pending: 0, complete: 1 }, _default: 0
```

Comment:
we have more statuses then 2

Reply:
I have another enum that show on which step the order is .